### PR TITLE
AOT test fixes

### PR DESCRIPTION
**Pull request by nirvdrum**

### CODE REVIEW

File: tool/jt.rb [hunk 1]
```diff
@@ -1134,7 +1134,8 @@ def test_specs(command, *args)
 
       options += %w[--excl-tag graalvm]
       options << '-t' << ENV['AOT_BIN']
-      options << '-T-XX:OldGenerationSize=2G'
+      options << '-T-XX:YoungGenerationSize=2G'
+      options << '-T-XX:OldGenerationSize=4G'
```
Comment (eregon) [inline tool/jt.rb]: Is 1G/2G like JS less optimal for specs?
2G of YoungGenerationSize will cause rather long young collections I would guess,

Comment (nirvdrum) [inline tool/jt.rb]: I can try that. Previously we were using 256 MB / 2 GB. Bumping them did increase collection time, but there were much fewer collections. With 2GB / 4GB, I see:

```
[Incremental GC (CollectOnAllocation.Sometimes) 2089337K->113907K, 0.3720967 secs]
[Full GC (CollectOnAllocation.Sometimes) 2203267K->102357K, 0.2234189 secs]
[Full GC (CollectOnAllocation.Sometimes) 2192367K->110995K, 0.2211100 secs] 
```

Comment (nirvdrum) [inline tool/jt.rb]: With 1GB / 2GB, I see:

```
[Incremental GC (CollectOnAllocation.Sometimes) 1044865K->69684K, 0.2061001 secs]
[Full GC (CollectOnAllocation.Sometimes) 1114444K->97982K, 0.2275266 secs]
[Incremental GC (CollectOnAllocation.Sometimes) 1143345K->119827K, 0.0537639 secs]
[Full GC (CollectOnAllocation.Sometimes) 1165246K->92144K, 0.1853570 secs]
[Incremental GC (CollectOnAllocation.Sometimes) 1137666K->124289K, 0.0797027 secs]
[Incremental GC (CollectOnAllocation.Sometimes) 1169413K->138856K, 0.0398659 secs]
[Incremental GC (CollectOnAllocation.Sometimes) 1184287K->168942K, 0.0797133 secs]
[Full GC (CollectOnAllocation.Sometimes) 1214399K->114031K, 0.2234832 secs]
```

Comment (nirvdrum) [inline tool/jt.rb]: I don't think it matters too much one way or another. If we have memory constraints somewhere, we can lower it.

**Pull request merged**